spot-ml/src/main/scala/org/apache/spot/SpotLDAWrapper.scala

```diff
@@ -0,0 +1,189 @@
+package org.apache.spot
```

Comment:
I thought about it, and wasn't sure. It seemed kind of fussy to me for two files. But your opinion breaks the tie.